add_new_column_to_a_materialized_view/README.md

```diff
@@ -9,9 +9,9 @@ This change needs to re-create the Materialized View and populate it again with
 For that the steps will be:
 
 1. Change the Materialized View (Pipe and Data Source) to add the new column.
-2. Bump version from 0.0.0 -> 0.0.1. It will create a new Preview Release internally forking the Materialized View and its dependencies.
-3. Backfill the Preview Release Materialized View with the data previous to its creation.
-4. Promote the release from Preview to Live.
+2. Bump version from 0.0.0 -> 1.0.0. Bumping the major version, it will create a new `Preview Release`` internally forking the Materialized View and its dependencies.
```

Comment:
```suggestion
2. Bump version from 0.0.0 -> 1.0.0. Bumping the major version, it will create a new `Preview Release` internally forking the Materialized View and its dependencies.
```